src/theme/src/hooks/useSegmentedControlAppearance.js

```diff
@@ -77,6 +77,7 @@ const getSegmentedControlStyles = theme => {
       }
     },
     [focusState]: {
+      zIndex: 1,
```

Comment:
I noticed that the focus style was funky when hovering other options in a segmented control. This helps it not look weird.

Reply:
I think this comes from `constants/StackingOrder` -- would avoid arbitrary `zIndex` throughout here.

Reply:
Hm, this is different than stacking order though – this is relative to sibling z-index. zIndex 1 is all you need. There is no other zIndex value?